rbac/addressing/addresser.py

```diff
@@ -75,8 +75,8 @@ class TaskRelationshipNamespace(enum.IntEnum):
 
 FAMILY_NAME = "rbac"
 FAMILY_VERSION = "1.0"
-NS = sha512(FAMILY_NAME.encode()).hexdigest()[:6]
-FAMILY_PREFIX = "9f4448"
+NAMESPACE = sha512(FAMILY_NAME.encode()).hexdigest()[:6]
+NS = NAMESPACE
```

Comment:
Can NS go away now?

Reply:
rbac/addressing/addresser.py
Disappears in the next PR